src/components/FeeLabelToggle/FeeLabelToggle.js

```diff
@@ -100,7 +100,7 @@ const FeeLabelToggle = ({
   isGasTokenSupported,
   hasError,
 }: Props) => {
-  const [isFiatValueVisible, setIsFiatValueVisible] = useState(showFiatDefault);
+  const [isFiatValueVisible, setIsFiatValueVisible] = useState(!showFiatDefault);
```

Comment:
This code change is incorrect, as it causes `showFiatDefault` prop to cause token value to be shown(!)

The correct solution would be to:
1) Remove `showFiatDefault` prop from this component and all it's use cases
2) Initialising `isFiatValueVisible` to `true`

Reply:
@mdjastrzebski Thanks for bringing this. This has been done.

Reply:
Please also remove `showFiatDefault` prop from all `FeeLabelToggle` use-cases (~14 files)

Reply:
Sorry. I checked with yarn run flow. It did not show any error so I missed this out. Done